distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestConfiguration.java

```diff
@@ -363,13 +129,45 @@ public boolean matches(Object object) {
             "ddf.test.itests.platform.TestPlatform.startup.invalid");
 
     @BeforeExam
-    @SuppressWarnings("unchecked")
     public void beforeExam() throws Exception {
+        basePort = getBasePort();
         getAdminConfig().setLogLevels();
         getServiceManager().waitForAllBundles();
         console = new KarafConsole(bundleCtx);
     }
+    
+    public void resetInitialState() throws Exception {
+
+        FileUtils.deleteQuietly(getExportDirectory().toFile());
+        FileUtils.deleteQuietly(new File(TEST_FILE));
+
+        FileUtils.cleanDirectory(getPathToProcessedDirectory().toFile());
+        FileUtils.cleanDirectory(getPathToFailedDirectory().toFile());
+
+        if (Files.exists(SYSTEM_PROPERTIES_COPY)) {
```

Comment:
Do you think it'd be worth extracting a private utility method since we repeat this logic 4 times?